site/docs/v1/tech/core-concepts/entity-management.adoc

```diff
@@ -0,0 +1,133 @@
+---
+layout: doc
+title: Entity Management
+description: An overview of FusionAuth Entity Management
+---
+
+:sectnumlevels: 0
+
+== Overview
+
+There are many use cases where it is helpful to model entities in addition to users. Examples might include devices, cars, computers, customers, companies, etc. +
+
+[%hardbreaks]
+
+_Enter Entities._ Entities allow you to model everything right from FusionAuth!
+
+* <<Features>>
+* <<Common Applications>>
+* <<Entity Types Homepage>>
+* <<Entity Homepage>>
+
+=== Features
+
+==== Scalability
+- FusionAuth Entity Management supports large volumes of Entities right out of the box.
+- This is especially helpful for Internet of Things (IOT) devices; FusionAuth scales right alongside them.
+
+==== Typecasting
+- Entities can have types.
+- For example, `locks`, `cars`, `companies`, `corporate divisions`, `computers`, `APIs`
+- Entity Types can help define permissions.
+- You are limited only by your business need and imagination!
+
+==== Permissions aware
+- Permissions can be assigned to each Entity Type.
+- Entities can share permissions with other Entities (In OAuth parlance, this synonymous with the client credentials grant).
```

Comment:
`Entities can share permissions with other Entities` -> `Entities can be granted permissions on other Entities`

`In OAuth parlance, this synonymous with the client credentials grant` -> `In OAuth terms, entities can initiate a  client credentials grant to obtain access to obtain access to other entites`